js/letters.js

```diff
@@ -37,11 +37,11 @@ var counter = {
 function countLetters(counter, sample_text) {
   // FIX ME
   let last = sample_text[sample_text.length - 1];
```

Comment:
These two lines should be after the base case. The base case should be the first thing in any recursive function.